libevmasm/LinkerObject.h

```diff
@@ -48,6 +48,7 @@ struct LinkerObject
 	struct FunctionDebugData
 	{
 		std::optional<size_t> bytecodeOffset;
+		std::optional<size_t> instructionIndex;
```

Comment:
Wouldn't `tagIndex` be more appropriate?

Reply:
not sure - it's an index into the list of instructions - maybe `instructionOffset` to make it clear that it is similar to `bytecodeOffset`?

Reply:
> it's an index into the list of instructions

Ah, okay. I'm indifferent about `instructionOffset` v/s `instructionIndex`.